x-pack/plugins/infra/public/pages/metrics/inventory_view/components/layout.tsx

```diff
@@ -49,6 +50,8 @@ interface LegendControlOptions {
   legend: WaffleLegendOptions;
 }
 
+const HOSTS_LINK_LOCAL_STORAGE_KEY = 'inventoryUI:hostsLinkClicked';
```

Comment:
I didn't export this one for the tests because there were some issues with the import when I tried to run the tests (locally and in the CI): [the error I got in the CI](https://buildkite.com/elastic/kibana-pull-request/builds/108903#01866e49-06a2-4d24-bdcb-da3eacc48dea)